### PR TITLE
Fixed the 3 Buttons+ removed the gradient.

### PR DESCRIPTION
**Pull request by Willi86**
Moral of the story: always follow instructions - :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -168,11 +168,16 @@ <h1>Hacker Escape Rooms</h1>
                     vos
                     omnino desit illud quo solo felicitatis libertatisque perficiuntur.</p>
             </div>
+            
             <div class="main__storyBox--button button darkBlue">
-                <a href="about-us.html">Read the full story</a> <!--Check font size and styling -->
+                <a href="about-us.html">
```
Comment (MatsJoh50) [inline index.html:172]: Swap line 173 with 172 to catch the whole div and thus adding the animation to the button and not only the text

**Review by MatsJoh50 — CHANGES_REQUESTED** [review]
For some reason the TopThree function dosn't work when I test it, and I cant see why when I try to look at the current main branch and the PR.

Future bug to solve after merg maby?

**Review by AnnaLenaO — APPROVED** [review]
This works. It is possible to delete Hoover.css and Hoover.scc.map. Only Hoover.scss can be left in src. In the file package.json at row 7 and 8 there are two things that show what can be written in the terminal to get the code in Hover.scss working. 
In the terminal, write either 
npm run sass:watch
or
npm run sass:compile
The code then automatically appears in main.css. 
I also pulled the branch and tested with all three files in src and that also works.

File: index.html (hunk 1 above)
Comment (AnnaLenaO) [reply index.html:172]: Sorry, I saw this now. We have been working with this at the same time :) 
I thought it was ment to be like this with the animation :)

**Review by MatsJoh50 — APPROVED** [review]

**Pull request merged**